examples/wordcount/src/main/java/com/hazelcast/jet/examples/wordcount/WordCount.java

```diff
@@ -72,18 +72,19 @@ public static void main(String[] args) throws Exception {
     /**
      * This code illustrates a few more things about Jet, new in 0.5. See comments.
      */
-    private void go() {
+    private void go() throws InterruptedException {
         try {
             setup();
-            System.out.print("\nCounting words... ");
+            System.out.println("\nCounting words... ");
             long start = System.nanoTime();
             Pipeline p = buildPipeline();
             Observable<Entry<String, Long>> observable = jet.getObservable(COUNTS);
             observable.toFuture(s -> s.collect(toMap(Entry::getKey, Entry::getValue)))
```

Comment:
it would be better to take the future, and then just print the values, rather than calling `thenAccept`